src/main/plugin/dcat2/index-fields/index.xsl

```diff
@@ -248,13 +249,30 @@
           </resolutionScaleDenominator>
         </xsl:for-each>
 
-
-
-
+        <!--Overview-->
         <xsl:variable name="overviews"
                       select="dcat:distribution/*[dct:format/*/skos:prefLabel = 'WWW:OVERVIEW' and dcat:accessURL != '']"/>
         <xsl:copy-of select="gn-fn-index:add-field('hasOverview', if (count($overviews) > 0) then 'true' else 'false')"/>
 
+        <!--DOWNLOADS-->
+        <xsl:for-each select="dcat:distribution">
+          <xsl:variable name="format" select="dct:format"></xsl:variable>
+          <xsl:copy-of select="gn-fn-index:add-field('format', $format)"/>
+          <xsl:variable name="protocol" select="$format-mimetype-mapping/entry[format=$format]/protocol"/>
+          <xsl:variable name="linkName" select="dct:title"/>
+          <xsl:variable name="description" select="dct:description"/>
+          <xsl:variable name="url" select="dcat:accessURL"/>
+
+          <link type="object">{
+            "protocol":"<xsl:value-of select="gn-fn-index:json-escape($protocol)"/>",
+            "url":"<xsl:value-of select="gn-fn-index:json-escape($url)"/>",
+            "name":"<xsl:value-of select="$linkName"/>",
+            "format":"<xsl:value-of select="$format"/>",
```

Comment:
mediaType is not indexed, is that on purpose? I mean, I think `format` is enough but I wasn't sure when reading the commit message

Reply:
Nice catch it's because i'm working in the data_dir and forgot to get it back.
Thanks